### DISCUSSION TITLE
iOS still crashes on startup

### DISCUSSION THREAD

**Discussion by Etheri0us** [ppy/osu#16044] [Q&A] [answered]
Crashes on startup on iPhone 8, I’ve tried restarting but didn’t change anything.
 Im unable to post logs today but if you still need them by tomorrow I can post them.

**Comment by frenzibyte** [comment]
Logs are definitely required, without them there's no way to tell what your issue is. If it was crashing for everyone then the release would've been pulled at that time.

**Reply by Etheri0us** [reply]
I will get them to you asap then

**Comment by wisdom0054** [comment]
How do I get osu IOS?

**Reply by bdach** [reply]
i would recommend starting with [reading the README document](https://github.com/ppy/osu#running-osu)

**Comment by lelxdme** [comment]
I do get the same problem too, on Ipad 2020 with  IpadOS 14.8.1
It just crash whenever I tried to start the game
version: 2021.1211.0
EDIT: Tried on my friends iPad pro 2018 and it crash on start too

**Comment by SupDos** [comment]
@frenzibyte, I'm getting the same issue too, but iTunes shows no logs in File Sharing
![iTunes_2021-12-12_11-21-28](https://user-images.githubusercontent.com/6813986/145708645-4eff9cfb-d348-4673-bdcd-72982c7f7001.png)

Other apps on that page do show their files (just to make sure it wasn't an issue with my iPad or iTunes not showing them), and on my iPhone where it does run osu!'s files show up on that page

Might not be of any help but here's a recording of it crashing https://streamable.com/jqkuhq

**Reply by frenzibyte** [reply]
Sounds like a native crash then. To retrieve the crash report, go to Settings -> Privacy -> Analystics & Improvements -> Analystics Data, and then look for the latest file with the name `osu.iOS` followed by the date of the crash.

**Comment by Etheri0us** [comment]
I actually found a solution everyone! At least it worked for me. Try updating to iOS 15 it solved my issues!

**Reply by peppy** [reply]
It's probably going to be related with the 120hz changes @frenzibyte. Think it will need some fallback logic for 14.x and earlier.

**Reply by frenzibyte** [reply]
Right, will look into it.

**Comment by Joppe27** [comment]
If you're still waiting on logs:
[osu.iOS-2021-12-12-170713.ips.beta.zip](https://github.com/ppy/osu/files/7699093/osu.iOS-2021-12-12-170713.ips.beta.zip)
Crashes on iPhone 6S, iOS 13.5

**Reply by peppy** [reply]
Thanks! We've found the issue causing this and it should be resolved in the next testflight release.

**Comment by frenzibyte** [comment, marked as answer]
iOS has now been working for the past two releases, please check them out.
